documents/motions/04.2021.1 - WCA Board.md

```diff
@@ -28,23 +28,23 @@ The WCA Board of Directors (or simply WCA Board) is the leadership team of the W
+3. Decisions of the WCA Board are based on majority voting in the WCA Board.
+4. The WCA Board's agenda is based on:
+   1. Vision and strategy documents for the WCA, updated on a periodical basis by the WCA Board.
    2. Input from Staff, Regional Organizations, local organization teams, Registered Speedcubers, external organizations, media, or outsiders.
-   3. Internal initiatives and agreements within the WCA Board of Directors.
-5. Communication with or within the WCA Board of Directors:
-   1. Communication to the WCA Board of Directors shall generally take place via the email address [board@worldcubeassociation.org](mailto:board@worldcubeassociation.org), but could also be initiated via other communication channels.
-   2. All communication on behalf of the WCA Board of Directors should include all Directors, e.g. as a carbon copy (CC) of the WCA Board of Directors email address in email, a paper copy, or online messages.
-   3. Communication within the WCA Board of Directors will be arranged internally with full logging of electronic communication.
+   3. Internal initiatives and agreements within the WCA Board.
+5. Communication with or within the WCA Board:
+   1. Communication to the WCA Board shall generally take place via the email address [board@worldcubeassociation.org](mailto:board@worldcubeassociation.org), but could also be initiated via other communication channels.
+   2. All communication on behalf of the WCA Board should include all Directors, e.g. as a carbon copy (CC) of the WCA Board's email address in email, a paper copy, or online messages.
+   3. Communication within the WCA Board will be arranged internally with full logging of electronic communication.
 6. Removal of Directors
    1. Directors will automatically be removed at the end of their term. See Bylaws 3.4 (Election and Term of Office).
    2. Directors may decide to resign before the end of their term. See Bylaws 3.6 (Resignation).
-   3. The WCA Board of Directors may decide to remove a Director according to Bylaws 3.7 (Removal). Before this final decision is made the WCA Board must first ask the WCA Ethics Committee for a non-binding advice on other possible solutions to the case leading to the possible removal.
+   3. The WCA Board may decide to remove a Director according to Bylaws 3.7 (Removal). Before this final decision is made the WCA Board must first ask the WCA Ethics Committee for a non-binding advice on other possible solutions to the case leading to the possible removal.
    4. The WCA Voting Members may decide to remove a Director according to Bylaws 3.7 (Removal).
-7. Removal of all the Board of Directors
-   1. WCA Voting Members can request a voting to the WCA Ethics Committee to remove all the Board of Directors according to Bylaws 3.7 (Removal).
+7. Removal of all the WCA Board
+   1. WCA Voting Members can request a voting to the WCA Ethics Committee to remove all the WCA Board according to Bylaws 3.7 (Removal).
```

Comment:
The Bylaws referenced in this motion are all off, _Election and Term of Office_ should be 3.5 instead of 3.4, _Removal_  should be 3.8 instead of 3.7 and so on. I assume it broke with the last Bylaws amendment?

Reply:
Thanks, resolved.